trajectory_msgs/JointTrajectory path
bool wait
---
string message